### DISCUSSION TITLE
Self hosted Version ?

### DISCUSSION THREAD

**Discussion by tamaskan** [WebCrateApp/webcrate#26] [Q&A] [answered]
Hi, ich würde es gerne als Docker-Container selbst hosten, konnte aber kein offizielles Image finden. Gibt es eine entsprechende Anleitung (Dockerfile,docker-compose) ?

MfG

**Comment by BetaHuhn** [comment, marked as answer]
Hey! (ich antworte auf Englisch wenn das oke ist)

There is currently not docker image available and it's not planned for the near future.

WebCrate uses a bunch of [Deta](https://deta.sh) services like [Base](https://docs.deta.sh/docs/base/about/) and Auth and thus can't be run on any server without a rewrite of the backend. WebCrate can only be run on [Deta Space](https://deta.space) which is like self-hosting it but without managing a server.

**Reply by tamaskan** [reply]
Okay, thank you very much for the quick Reply :-)

**Reply by BetaHuhn** [reply]
No problem :)

**Reply by shyim** [reply]
So there is no real self-hosting planned? 

`which is like self-hosting it but without managing a server` that has nothing to do with self-hosting.

**Reply by BetaHuhn** [reply]
Self-hosting in this regard means that the data/compute stays with you and not me the developer. The app itself still runs on a provider, in this case [Deta Space](https://deta.space) and not on your own server.

**Reply by boexler** [reply]
I would also like to see a way to run it in a Docker container.
